recipes/virtual_dataframe/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set BASENAME = "virtual_dataframe" %}
+{% set PRJ_VERSION = "v0.1rc1" %}
+{% set SETUPTOOLS_SCM_PRETEND_VERSION = PRJ_VERSION %}
+{% set PRJ_PACKAGE = BASENAME %}
+{% set PYTHON_VERSION = "3.8" %}
+{% set PYTHON_VERSION_MAX = "3.9" %}
```

Comment:
This is not used

Reply:
Is true for PRJ_PACKAGE and PYTHON_VERSION_MAX.

The variable SETUPTOOLS_SCM_PRETEND_VERSION is used to 'force' the version when the build process is running outside of the original git directory [see](https://pypi.org/project/setuptools-scm/#:~:text=SETUPTOOLS_SCM_PRETEND_VERSION%3A-,when%20defined%20and%20not%20empty%2C%20its%20used%20as%20the%20primary%20source%20for%20the%20version%20number%20in%20which%20case%20it%20will%20be%20a%20unparsed%20string).

The PYTHON_VERSION is use in line 39, to accept to install this package only with python>3.8.